src/i18n/README.md

```diff
@@ -10,6 +10,13 @@ Translations all live in this GitHub repository. You can add and update them by
 ### How can I find out what needs translating?
 See our automated [Translation Status Overview issue](https://github.com/withastro/docs/issues/438) for a quick list of which pages are missing or need updating.
 
+
+> **Please do not translate any pages without first checking the status on our Tracker**! 
```

Comment:
We say “Translation Status Overview issue” above and “Tracker” here — maybe it’s not obvious those are the same thing? I’d suggest repeating similar language and another link wouldn’t hurt.

We could also use [GitHub’s syntax for a warning block](https://github.com/github-community/community/discussions/16925) here:

```suggestion
> **Warning**
> Please do not translate any pages without first checking their status in our [Overview Issue](https://github.com/withastro/docs/issues/438)! 
```